app/views/search/_account_account_user_link.html.haml

```diff
@@ -1 +1 @@
-= link_to "#{user.last_name}, #{user.first_name}", new_account_account_user_path(@account, :user_id => user.id)
\ No newline at end of file
+= link_to user.last_first_name, new_account_account_user_path(@account, :user_id => user.id)
```

Comment:
Thanks for catching this. I was doing a bunch of find/replace for these `_link` files. Guess I missed the hash style here.